ansible/roles/kubernetes/tasks/gen_certs.yml

```diff
@@ -51,3 +51,17 @@
     - "{{ kube_cert_dir }}/server.key"
     - "{{ kube_cert_dir }}/kubecfg.crt"
     - "{{ kube_cert_dir }}/kubecfg.key"
+
+- name: Remove CA private key if ca_keep_private_key is false
+  file:
+    path={{ kube_cert_dir}}/ca.key
+    state=absent
+  when: not ca_keep_private_key
+
+- name: Check CA private key permissions if ca_keep_private_key is True
+  file:
+    path={{ kube_cert_dir }}/ca.key
```

Comment:
You should use `:` instead of `=`.
That is currently not 100% enforced in all of the files here in contrib/ansible but from my point of view should be as there should be only **one** style for args.